### PR TITLE
Manual integration Tests

### PR DESCRIPTION
**Pull request by katelynsills**
This PR doesn't replace the automated integration tests that existed on Circle CI, but it does allow for a manual testing process. 

The automated integration tests would take your usual test and transform it into the different browser contexts, so that you were actually running all of your tests against Nat as bundled by Webpack, browserify, etc. Furthermore, this was set up to run in CI. 

This PR enables a manual version. Because we've moved to AVA and AVA is not supported in a browser environment, this version uses Nat tests which are manually copied and transformed back to tape. Then, you can run:

```
./integration-test/scripts/build-pre-release-test.sh
```

and visit the following pages in your browser, checking the console output:

`/integration-test/pre-release-browser-tests/mock-unpkg-umd/index.html`
`/integration-test/pre-release-browser-tests/webpack/index.html`
`/integration-test/pre-release-browser-tests/rollup/index.html`
`/integration-test/pre-release-browser-tests/browserify/index.html`

Related to #142, but doesn't solve it entirely

### CODE REVIEW

File: integration-test/transform-tests/config/rollup.config.cjs.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default [
     plugins: [
       replace({
         delimiters: ['', ''],
-        "import Nat from '../src/index';": "import Nat from '@agoric/nat';",
+        "import { isNat, Nat } from '../src/index';": "import { isNat, Nat } from '@agoric/nat';",
```
Comment (michaelfig) [inline integration-test/transform-tests/config/rollup.config.cjs.js:15]: Could this be more robust?

```suggestion
        "'../src/index'": "'@agoric/nat'",
```

File: integration-test/transform-tests/config/rollup.config.esm.js [hunk 2]
```diff
@@ -12,7 +11,7 @@ export default [
     plugins: [
       replace({
         delimiters: ['', ''],
-        "import Nat from '../src/index';": "import Nat from '@agoric/nat';",
+        "import { isNat, Nat } from '../src/index';": "import { isNat, Nat } from '@agoric/nat';",
```
Comment (michaelfig) [inline integration-test/transform-tests/config/rollup.config.esm.js:14]: Likewise:
```suggestion
        "'../src/index'": "'@agoric/nat'",
```

**Review by michaelfig — APPROVED** [review]
LGTM!

File: integration-test/transform-tests/config/rollup.config.cjs.js (hunk 1 above)
Comment (katelynsills) [reply integration-test/transform-tests/config/rollup.config.cjs.js:15]: Yup, that is better!

**Pull request merged**